webcompat/views.py

```diff
@@ -231,7 +231,8 @@ def create_issue():
             ip=request.remote_addr,
             url=form['url'].encode('utf-8')))
         # Checking blacklisted domains
-        if is_blacklisted_domain(urlparse.urlsplit(form['url']).hostname):
+        if form.get('submit_type') == PROXY_REPORT and \
```

Comment:
It's better to avoid the continuation character `\` in general. 
Your idea is good! but we could do that instead:

```python
        # Logging the ip and url for investigation
        log.info('{ip} {url}'.format(
            ip=request.remote_addr,
            url=form['url'].encode('utf-8')))
        # Check if the form is valid
        if not is_valid_issue_form(form):
            abort(400)
        # Anonymous reporting
        if form.get('submit_type') == PROXY_REPORT:
            # Checking blacklisted domains
            domain = urlparse.urlsplit(form['url']).hostname
            if is_blacklisted_domain(domain):
                msg = app.config['IS_BLACKLISTED_DOMAIN'].format(form['url'])
                flash(msg, 'notimeout')
                return redirect(url_for('index'))
            json_response = report_issue(form, proxy=True)
            session['show_thanks'] = True
            return redirect(
                url_for('show_issue', number=json_response.get('number')))
        # Authenticated reporting
        if form.get('submit_type') == AUTH_REPORT:
```

What do you think?

Reply:
Good idea. I will make the changes